swift/tools/tracing-config.lua

```diff
@@ -3,6 +3,53 @@ function RegisterExtractorPack(id)
     local relativeSwiftExtractor = extractorDirectory .. 'extractor'
     local swiftExtractor = AbsolutifyExtractorPath(id, relativeSwiftExtractor)
 
+    function indexOf(array, value)
+      for i, v in ipairs(array) do
+        if v == value then
+          return i
+        end
+      end
+      return nil
+    end
+
+    -- removes upsupported CLI arg including the following how_many args
+    function strip_unsupported_arg(args, arg, how_many)
+      local index = indexOf(args, arg)
```

Comment:
nit: you can make `how_many` have a default of 0, like this:
```suggestion
      how_many = how_many or 0
      local index = indexOf(args, arg)
```
then if you cann the function without the third argument it will use 0 (thechnically, it passes `nil` to the parameter).